src/ao-profile/profile.lua

```diff
@@ -343,156 +339,120 @@ Handlers.add('Credit-Notice', Handlers.utils.hasMatchingTag('Action', 'Credit-No
 		end
 	end)
 
--- Data - { Id, Quantity }
+-- Tag - { Quantity = # } - Assignment from Spawn, msg.Id is the asset ID
 Handlers.add('Add-Uploaded-Asset', Handlers.utils.hasMatchingTag('Action', 'Add-Uploaded-Asset'),
 	function(msg)
-		-- local authorizeResult, message = authorizeRoles(msg)
-		-- if not authorizeResult then
-		--     ao.send(message)
-		--     return
-		-- end
+		local reply_to = msg.Target or msg.From
```

Comment:
is msg.Target sometimes missing/blank?

Reply:
In our cases, I don't think so, but maybe in others...Tom mentioned this may be blank. In that case, there should be a msg.from